### DISCUSSION TITLE
docs/ninjutsu/medium-ranking-ninja/fake-windows-update

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#24] [General]
# docs/ninjutsu/medium-ranking-ninja/fake-windows-update

没错！这是一款“高效工作”的软件！！

https://996.ninja/docs/ninjutsu/medium-ranking-ninja/fake-windows-update

**Comment by wududu0920** [comment]
那个假装更新的软件我至少在14年就知道有类似的了哈哈哈哈哈

**Reply by MingGH** [reply]
是的，有些软件都挺老的了
